test/lib/generate/editor-config-test.ts

```diff
@@ -0,0 +1,51 @@
+import { getEndOfLine } from '../../../lib/string.js';
+import mockFs from 'mock-fs';
+import { jest } from '@jest/globals';
+
+describe('string (getEndOfLine)', function () {
```

Comment:
Can you add a test in here with an empty `.editorconfig` and also a non-existent `.editorconfig`?

Reply:
Sure, do you want me to remove the test from `string-test.ts` named `handles when .editorconfig is not available and fallbacks to 'EOL' from 'node:os'` as it will overlap with the non-existent `.editorconfig` test?

Reply:
I think it's fine to have both since one is a unit test and one is an integration test.